[BUG] mvn clean prints an error message on a clean dir

**Describe the bug**
The error is printed when some of the target dirs are missing 
```
mvn -q clean
[ERROR]      [exec] Result: 1
```

we need to add -f to https://github.com/NVIDIA/spark-rapids/commit/f2f55ae74eb67857c7dd3197442807e581ed40ee#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R1112
**Steps/Code to reproduce bug**
```
mvn clean
```

**Expected behavior**
repeated clean should not produce error messages

**Environment details (please complete the following information)**
 local

--- a/pom.xml
+++ b/pom.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  Copyright (c) 2020-2021, NVIDIA CORPORATION.
+  Copyright (c) 2020-2022, NVIDIA CORPORATION.
 
   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
@@ -1323,18 +1323,12 @@
                             <target>
                                 <dirset dir="${project.basedir}" includes="**/target" id="target.dirs.for.clean"/>
                                 <pathconvert property="target.dirs.str" pathsep=" ">
-                                    <sort>
-                                        <dirset refid="target.dirs.for.clean"/>
-                                        <!-- remove subdirs first if we have something like target/somedir/target -->
-                                        <reverse xmlns="antlib:org.apache.tools.ant.types.resources.comparators">
-                                            <name/>
-                                        </reverse>
-                                    </sort>
+                                   <dirset refid="target.dirs.for.clean"/>
                                 </pathconvert>
                                 <echo>Cleaning build directories of all modules ${target.dirs.str}</echo>
                                 <!-- workaround ant delete does not work with dirset -->
                                 <exec dir="${project.basedir}" executable="rm">
-                                    <arg value="-r"/>
+                                    <arg value="-rf"/>
                                     <arg line="${target.dirs.str}"/>
                                 </exec>
                             </target>

--- a/sql-plugin/pom.xml
+++ b/sql-plugin/pom.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  Copyright (c) 2020-2021, NVIDIA CORPORATION.
+  Copyright (c) 2020-2022, NVIDIA CORPORATION.
 
   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
@@ -366,10 +366,6 @@
                   </configuration>
                 </execution>
               </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>build-helper-maven-plugin</artifactId>
             </plugin>
             <plugin>
                 <groupId>net.alchim31.maven</groupId>

